test/catch-override-exception.test.js

```diff
@@ -1,26 +1,26 @@
 'use strict'
 
-const { test } = require('tap')
+const { test } = require('node:test')
 const boot = require('..')
 
-test('catch exceptions in parent.override', (t) => {
-  t.plan(2)
+test('catch exceptions in parent.override', (t, testCompleted) => {
+  t.plan(1)
 
   const server = {}
 
   const app = boot(server, {
     autostart: false
   })
   app.override = function () {
-    throw Error('catch it')
+    throw new Error('catch it')
```

Comment:
```suggestion
    throw Error('catch it')
```

Reply:
throw an error without `new` keyword?

Reply:
There is no different with or without the `new`.
So, the change itself is unnecessary.